src/components/pages/Home/RenderHomePage.js

```diff
@@ -6,11 +6,11 @@ function RenderHomePage(props) {
   const { userInfo, authService } = props;
   return (
     <div>
-      <h1>Hi {userInfo.name} Welcome to Labs Basic SPA</h1>
+      <h1>Hi {userInfo.name} Welcome to Express Groomer!</h1>
```

Comment:
You need to update the content of the test file too after changing this: https://github.com/Lambda-School-Labs/LabsPT12-Express_Groomer-TeamE-FE/blob/main/src/__tests__/RenderHomePage.test.js#L20

Reply:
Thank you for the help!